.eslintignore

```diff
@@ -0,0 +1,3 @@
+# Ignore all files except webpack directory
+**/*.js
+!webpack/**/*.js
```

Comment:
I guess you just want to avoid the assets js files?
I think you can instead ignore what's in the `app` folder:

```suggestion
app/**
```